### PR TITLE
HWKINVENT-90 | Metric data type of MetricType

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Comment by metlos** [comment]
Great job!

File: hawkular-integrated-inventory-rest/src/main/java/org/hawkular/integrated/inventory/TemporaryHacks.java [hunk 1]
```diff
@@ -58,11 +59,13 @@ private void createTenantMetadata(Tenant tenant, Inventory inventory) {
         inventory.inspect(tenant).resourceTypes().create(ResourceType.Blueprint.builder().withId("URL").build());
         Log.LOGGER.autoCreatedEntity("resource type", "URL", tenant.getId());
 
-        inventory.inspect(tenant).metricTypes().create(MetricType.Blueprint.builder().withId("status.code.type")
+        inventory.inspect(tenant).metricTypes()
+                .create(MetricType.Blueprint.builder(MetricDataType.COUNTER).withId("status.code.type")
```
Comment (pilhuhn) [inline hawkular-integrated-inventory-rest/src/main/java/org/hawkular/integrated/inventory/TemporaryHacks.java:63]: This is certainly not a counter. This is the http status code.

File: hawkular-integrated-inventory-rest/src/main/java/org/hawkular/integrated/inventory/TemporaryHacks.java [hunk 2]
```diff
@@ -58,11 +59,13 @@ private void createTenantMetadata(Tenant tenant, Inventory inventory) {
...
                 .withUnit(MetricUnit.NONE).build());
         Log.LOGGER.autoCreatedEntity("metric type", "status.code.type", tenant.getId());
 
-        inventory.inspect(tenant).metricTypes().create(MetricType.Blueprint.builder().withId("status.duration.type")
+        inventory.inspect(tenant).metricTypes()
+                .create(MetricType.Blueprint.builder(MetricDataType.AVAILABILITY).withId("status.duration.type")
```
Comment (pilhuhn) [inline hawkular-integrated-inventory-rest/src/main/java/org/hawkular/integrated/inventory/TemporaryHacks.java:68]: Duration is a gauge

**Comment by pilhuhn** [comment]
How can e.g. the agent supply that metric type? Does @jmazzitelli know about it?

**Comment by pavolloffay** [comment]
Do you mean create another DataMetricType?

**Comment by pilhuhn** [comment]
Less create a new one , but rather supply it when creating/updating a metric definition

**Pull request merged**